Source/WebCore/editing/CompositeEditCommand.cpp

```diff
@@ -453,9 +453,9 @@ RefPtr<DataTransfer> CompositeEditCommand::inputEventDataTransfer() const
 EditCommandComposition* CompositeEditCommand::composition() const
 {
     for (RefPtr command = this; command; command = command->parent()) {
-        if (auto composition = command->m_composition) {
+        if (auto* composition = command->m_composition.get()) {
             ASSERT(!command->parent());
-            return composition.unsafeGet();
+            return composition;
```

Comment:
This doesn't seem like an improvement.  What guarantees the lifetime of the returned object?

Reply:
We discussed this offline, and this seems fine since the caller is responsible for retaining the return value, and this is basically a getter that walks its parents to find a value to return.